loginPage.js

```diff
@@ -0,0 +1,212 @@
+//assigned html buttons
+const loginPopUpButton=document.getElementById("loginPopUpButton");
+const loginButton=document.getElementById("loginButton");
+const signUpButton=document.getElementById("signupButton");
+
+//taking values from signup form when creating an account
+const signupPopupForm=document.querySelector(".signUpPopup");
+const nameSignUp=document.getElementById("nameSignup");
+const mobileNumSignUp=document.getElementById("mobileNumSignUp");
+const mailIdSignUp=document.getElementById("mailIdSignUp");
+const passwordSignUp=document.getElementById("passwordSignUp");
+
+//taking values from login form
+const loginPopUpForm=document.getElementById('popupForm');
+const mailIdLogin=document.getElementById("mailIdLogin");
+const passwordLogin=document.getElementById("passwordLogin");
+
+//Alert pop up
+const alertPopUp=document.getElementById("alertPopup");
+const successAlert=document.getElementById("alertSuccess");
+
+//array to store user data as objects
+let storeUserData=[];
+
+////password encrypt and decrypt
+const crypt = {
+    // THE SECRET KEY
+    secret : "CIPHERKEY",
+   
+    //ENCRYPT
+    encrypt : clear => {
+      let cipher = CryptoJS.AES.encrypt(clear, crypt.secret);
+      return cipher.toString();
+    },
+   
+    //DECRYPT
+    decrypt : cipher => {
+      let decipher = CryptoJS.AES.decrypt(cipher, crypt.secret);
+      return decipher.toString(CryptoJS.enc.Utf8);
+    }
+  };
+
+//when the login popup button clicked
+loginPopUpButton.addEventListener('click',()=>{
+   document.querySelector(".backgroundBlur").style.display="block"
+   loginPopUpForm.style.display="block";
+})
+
+//when login button clicked
+loginButton.addEventListener('click',()=>{
+    if(mailIdLogin.value =="" || passwordLogin.value=="" ){
+        alertPopUp.style.display="block";
+    }else{
+         loginValidate();
+         resetLogin();
+    }
+});
+
+//validate login
+function loginValidate(){
+    let testEmail=/^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/g.test(mailIdLogin.value);
+    let testPassword=/^(?=.{8,})(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?!.*\s).*$/gm.test(passwordLogin.value);
+    if(testEmail && testPassword){
+        let userInfo=storeUserData.find((user)=>user.mailId==mailIdLogin.value);
+        if (userInfo) {
+            // Decrypt the password
+            let decryptedPassword = crypt.decrypt(userInfo.password);
+            if(!decryptedPassword==passwordLogin.value){
+                alert("Incorrect password!")
+            }else{
+                window.location.href="/registrationPAge/dashBoard.html"
+                setCookie('Username',userInfo.name,1);
+            }
+            
+        } else {
+            alert("User not found");
+        }
+    }else{
+        alert("Invalid Input!")
+    }
+}
+
+//set cookie
+function setCookie(userName,userValue,expDays){
+    const d=new Date();
```

Comment:
naming

Reply:
should i correct it to 
cost date=new Date();

Reply:
yes

Reply:
done